README.md

```diff
@@ -0,0 +1,179 @@
+# :bridge_at_night: Zuul MQTT reporter -> Matrix chat bridge
+
+Relay MQTT reports from [Zuul-CI](https://zuul-ci.org/) to the [Matrix chat](https://matrix.org/).
+
+Currently, Zuul-CI has the capability to distribute build reports to various supported platforms
+(refer to the supported drivers [list](https://zuul-ci.org/docs/zuul/latest/drivers/index.html)).
+However, there is currently no built-in support for conveniently sending these build reports to the Matrix chat.
+
+This project offers a straightforward solution for receiving build reports from Zuul's MQTT reporter and,
+using predefined template, forwarding these messages to a Matrix chat room.
+
+![arch](./docs/images/bridge.png)
+
+## Usage
+
+This section describes a method for integrating the zuul-mqtt-matrix-bridge project into your Zuul-CI infrastructure. It assumes that your Zuul infrastructure is running within Docker containers on the same Docker network. If your setup is different, you can still follow this guide, but make sure that your Zuul deployment can access the MQTT broker container.
+
+### MQTT Broker configuration
+
+The MQTT broker serves as the backend system responsible for handling messages between the Zuul MQTT client, acting as a publisher, and the zuul-mqtt-matrix-bridge client, acting as a subscriber. The [docker-compose](./docker/docker-compose.yaml) file deploys the Eclipse Mosquitto server, an open-source message broker that utilizes the MQTT protocol.
+
+You can explore a basic MQTT broker configuration file, [mosquitto.conf](./docker/mosquitto_config/mosquitto.conf), which ensures broker persistence and enforces client authentication via a password file. A password file is a straightforward method for storing usernames and their corresponding encrypted passwords in a single file. Additionally, an example [password file](./docker/mosquitto_config/pwfile) is provided, already containing a user record for the username `mqtt-user` with the password `secret`. 
```

Comment:
```suggestion
You can explore a basic MQTT broker configuration file, [mosquitto.conf](./docker/mosquitto_config/mosquitto.conf), which ensures broker persistence and enforces client authentication via a password file. A password file is a straightforward method for storing usernames and their corresponding encrypted passwords in a single file. Additionally, an example [password file](./docker/mosquitto_config/pwfile) is provided, already containing a user record for the username `mqtt-user` with the password `secret`. You are strongly advised to change these credentials.
```

it would be even better if we can omit credentials in examples, because they tend to end up in live systems somehow, for real life vulnerabilities that follow from this see e.g.: https://arxiv.org/pdf/1704.02786.pdf